lib/kwtool/crawlers.ex

```diff
@@ -0,0 +1,28 @@
+defmodule Kwtool.Crawlers do
+  import Ecto.Query, warn: false
+
+  alias Kwtool.Accounts.Schemas.User
+  alias Kwtool.Crawlers.Schemas.Keyword
+  alias Kwtool.Crawlers.UploadParser
+  alias Kwtool.Repo
+
+  def save_keywords_list(keyword_file, %User{} = user) do
+    keyword_file
+    |> UploadParser.load_from_file()
+    |> Enum.each(fn keyword ->
+      create_keyword(%{
+        phrase: List.first(keyword),
+        status: 0,
```

Comment:
This default status could be omitted either via a create_changeset and/or a custom method:

```elixir
def create_changeset(keyword \\ %__MODULE__{}, attrs)
  keyword
  |> cast(attrs, [:phrase, :user_id])
  |> validate_required([:phrase, :user_id])
  |> put_change(:0, 0)
end
```
```elixir
def create_keyword(phrase, user_id)
   %Keyword{}
   |> Keyword.create_changeset(%{phrase: phrase, user_id: user_id})
   |> Repo.insert()
end
```

In any case, if status = 0 is the expected default status then this default value should be set at the database level. It's part of data integrity.

Reply:
yes, good suggestion.
I refactor the `status` from `int` to `string` for more meaning.